recipes/blas_openblas/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: blas_openblas
+  version: 1.0.0
+
+build:
+  number: 0
+  track_features:
+    - blas_openblas
+
+test:
+  commands:
+    - conda list blas_openblas
```

Comment:
So, this is a challenging package to test because it doesn't do much, but I tried to test it anyways and this seems like a simple test. However, it fails because it has not activated the `_test` environment. Here is the full error message.

```
TEST START: blas_openblas-1.0.0-0
Fetching package metadata: ........
.Solving package specifications: .........

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    blas_openblas-1.0.0        |                0          634 B  file:///opt/conda/conda-bld/linux-64/

The following NEW packages will be INSTALLED:

    blas_openblas: 1.0.0-0 file:///opt/conda/conda-bld/linux-64/

+ conda list blas_openblas
Error: Error: environment does not exist: /opt/conda/envs/_build
#
# Use 'conda create' to create an environment before listing its packages.
TESTS FAILED: blas_openblas-1.0.0-0
```